### PR TITLE
Hack `jsonrpsee`, `substrate-subxt`, Workround For #7

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -25,6 +25,8 @@ name = "bridger"
 path = "src/bin/bridger.rs"
 
 [dependencies]
+async-std = "1.6.5"
```
Comment (clearloop) [inline Cargo.toml:28]: Does not need `async-std`, tokio in this repo offers `sleep` as well.

File: src/cmd/run.rs [hunk 2]
```diff
@@ -7,23 +7,16 @@ use crate::{
...
-        } else {
-            std::env::set_var("RUST_LOG", "info");
-        }
-    }
-    env_logger::init();
-
+pub async fn exec(path: Option<PathBuf>) -> Result<()> {
     // config
-    let config = Config::new(path)?;
+    let config = Config::new(path.clone())?;
```
Comment (clearloop) [inline src/cmd/run.rs:19]: Why `.clone()` here?

File: src/cmd/run.rs [hunk 3]
```diff
@@ -33,6 +26,7 @@ pub async fn exec(path: Option<PathBuf>, verbose: bool) -> Result<()> {
     // APIs
     let shadow = Arc::new(Shadow::new(&config));
     let darwinia = Arc::new(Darwinia::new(&config).await?);
+    let killer = darwinia.client.rpc.client.killer.clone();
```
Comment (clearloop) [inline src/cmd/run.rs:29]: Why not `let killer = &darwinia.client.rpc.client.killer;` ?

**Review by HackFisher — APPROVED** [review]

File: src/cmd/run.rs (hunk 2 above)
Comment (AurevoirXavier) [reply src/cmd/run.rs:19]: forget to remove

**Pull request merged**